R/paged.R

```diff
@@ -148,7 +148,7 @@ html_format = function(
     }
   }
 
-  css2 = grep('[.]css$', css, value = TRUE, invert = TRUE)
+  css2 = grep('[.]s?css$', css, value = TRUE, invert = TRUE)
```

Comment:
To be complete, the regex would be 
```suggestion
  css2 = grep('[.](?:sa|sc|c)ss$', css, value = TRUE, invert = TRUE)
```

Reply:
Ok. I will test the changes and update the docs.